StoreApp.BLL/DTO/FilterProductsDTO.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
Do you need to keep all these namespaces?

Reply:
fixed